text/0000-async_await.md

```diff
@@ -238,7 +238,7 @@ fn foo<'a>(arg1: &'a str, arg2: &str) -> impl Future<Output = usize> + 'a {
     // do some initialization using arg2
 
     // closure which is evaluated immediately
-    async move {
+    move async {
```

Comment:
The RFC should specify the order of `async` and `move` keywords for async blocks and closures.

`async move` is my preference. @cramertj (https://github.com/rust-lang/rfcs/pull/2394#discussion_r183189740) and @Centril (https://github.com/rust-lang/rfcs/pull/2394#discussion_r183190801) like it as well
@rpjohnst wants consistent ordering (https://github.com/rust-lang/rfcs/pull/2394#discussion_r183189953)

Reply:
`async move` looks better imo, and more in line with `const unsafe async move` idea proposed by @cramertj 

It's possible that having `move` first is more consistent, but consider that `async` did not exist back then when move was introduced and if you write it with a macro it will be `async!(move || ...`.

Just 0.02€